src/installer.ts

```diff
@@ -141,6 +114,51 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+async function installNodeVersion(
+  info: INodeVersionInfo,
+  auth: string | undefined
+): Promise<string> {
+  const localPath = tc.find('node', info.resolvedVersion);
+  if (localPath) {
+    console.log(`Found in cache @ ${localPath}`);
```

Comment:
Instead of `console.log`, you can use `core.info` for tracing. It's effectively the same thing but it's just fits better with the rest of the code. 

For some reason it isn't documented, a PR is on the way to update the README
https://github.com/actions/toolkit/blob/9e2d61e548a6edd163b05e31b69a8639417cdcac/packages/core/src/core.ts#L153

Reply:
Just copy-pasted existing logging. Not sure why it was done initially. I have replaced console.log to core.info through all code